### ISSUE TITLE
There is no region field in Configuration spec

### ISSUE THREAD

**Issue by barnettZQG** [kubevela/kubevela#3900]
**Describe the bug**

In VelaUX, users could set the region in target, and the application with cloud service could be deployed to the region defined in every target. 

```
spec:
  components:
  - name: mysql-rds
    properties:
      account_name: yueda
      allocate_public_connection: true
      database_name: yueda
      deleteResource: true
      instance_name: yueda
      password: yueda123
      privilege: ReadWrite
      vswitch_id: ""
    type: alibaba-rds
  policies:
  - name: env-bindings-default
    properties:
      envs:
      - name: default
        patch:
          components:
          - name: mysql-rds
            properties:
              providerRef:
                name: aliyun-yueda
              region: cn-beijing
              writeConnectionSecretToRef:
                name: mysql-rds-default
                namespace: default
            type: alibaba-rds
        placement:
          clusterSelector:
            name: local
          namespaceSelector:
            name: default
    type: env-binding
```
Currently, the region field in component properties does not take effect. Others, we can not get the region from the Configuration.

**Expected behavior**
Support set the region in component properties and could get the region from the Configuration.

**Screenshots**

![image](https://user-images.githubusercontent.com/18493394/168529487-e851498c-fac8-4d15-a836-b1c22ed022d8.png)

**KubeVela Version**

1.3.4

**Comment by zzxwill** [comment]
Use `customRegion` please due to the issue https://github.com/kubevela/kubevela/pull/3503.

**Comment by wonderflow** [comment]
```
spec:
  components:
  - name: mysql-rds
    properties:
      customRegion: xx
      ...
    type: alibaba-rds
```

=> region align with provider by default, you can also set it in `properties.customRegion`,  it should be reported in `status.region`

```
kind: Configuration
spec:
   ...
status:
  region: xxx
```

**Comment by barnettZQG** [comment]
Fixes by kubevela/velaux#538
